_posts/2021-08-29-mosaic-fusion.md

```diff
@@ -0,0 +1,100 @@
+> Your block function gets information about where it is in the array by accepting a special `block_info` or `block_id` keyword argument.
+
+The basic outline of the image mosiac workflow is as follows:
+1. Read in image tiles acquired from the microscope.
+2. Calculate the location of the image tiles, using information from the sample stage position at the time each image tile was acquired.
+3. Use an affine transformation to place the image tiles in their proper location.
+4. Fuse the combined images into a single result.
+
+
+### Results
+
+Using the dask-based image fusion method, we could achieve a speed up from several hours to tens of minutes compared to 
+a previous method based on an ImageJ plugin for datasets with many image tiles (~500-1000 tiles) on similar workstation hardware.
+Due to dask's ability to handle data out-of-core and chunked array storage using zarr it is also possible to run the 
+fusion on hardware with limited RAM.
+
+The image below shows mosaic image fusion for a smaall examples with 63 image tiles, displayed in the napari image viewer.
+
+![Mosaic fusion images in the napari image viewer](/images/mosaic-fusion/NapariMosaics.png)
+
+And here is an animation of whole slide mosaic fusion.
+
+![Animation of whole slide mosaic fusion images](/images/mosaic-fusion/Lama_whole_slide.gif)
+
+### Code
+
+Code relatiing to this mosaic image fusion project can be found in the `DaskFusion` GitHub repository here: https://github.com/VolkerH/DaskFusion
+
+There is a self-contained example available in [this notebook](https://github.com/VolkerH/DaskFusion/blob/main/Load_Mosaic.ipynb), which downloads reduced-size example data to demonstrate the process.
+
+
+
+
+## What's next?
+
+Currently, the DaskFusion code is a proof of concept for single-channel 2D images and simple maximum projection for blending the tiles in overlapping areas, it is not production code.
+However, the same principle can be used for fusing multi-channel image volumes,
+such as from Light-Sheet data if the tile chunk intersection calculation is extended to higher-dimensional arrays.
+Such even larger datasets will benefit even more from leveraging dask,
+as the processing can be distributed across multiple nodes of a HPC cluster using [dask jobqueue](http://jobqueue.dask.org/en/latest/).
+
+### Also see
+Marvin's lightning talk on multi-view image fusion is up online now: https://www.youtube.com/watch?v=YIblUvonMvo&list=PLJ0vO2F_f6OBAY6hjRHM_mIQ9yh32mWr0&index=10
+
+The GitHub repository [MVRegFus](https://github.com/m-albert/MVRegFus) that Marvin talks about in the video is available here: https://github.com/m-albert/MVRegFus
+
+
+## Acknowledgements
+
+The sample images are courtesy of M. Shahraz, EMBL Alexandrov Lab.
```

Comment:
Let's put all of the acknowledgements about who did what in one place. I don't mind where, either at the top where the others currently are, or we could move it all down to this section instead.